modules/connectivity/ConnectionQuality.js

```diff
@@ -61,32 +34,32 @@ let startBitrate = 800;
  * @param simulcast {boolean} whether simulcast is enabled or not.
  * @param resolution {Resolution} the resolution.
  * @param millisSinceStart {number} the number of milliseconds since sending video started.
- * @param videoQualitySettings {Object} the bitrate and codec settings for the local video source.
+ * @param bitrates {Object} the bitrates for the local video source.
  */
-function getTarget(simulcast, resolution, millisSinceStart, videoQualitySettings) {
+function getTarget(simulcast, resolution, millisSinceStart, bitrates) {
     let target = 0;
     let height = Math.min(resolution.height, resolution.width);
 
     // Find the first format with height no bigger than ours.
-    let simulcastFormat = kSimulcastFormats.find(f => f.height <= height);
+    let qualityLevel = VIDEO_QUALITY_LEVELS.find(f => f.height <= height);
```

Comment:
The only case which would lead to qualityLevel being undefined here would be something < 180. Does that make any sense? Even then, wouldn't pretending it's 180 be ok?

Reply:
Techincally, the resolution won't be less than 180 since its coming from here https://github.com/jitsi/lib-jitsi-meet/blob/master/service/RTC/Resolutions.js#L1. 
Even otherwise, I have added `VIDEO_QUALITY_LEVEL.NONE` with a bitrate of 0 to handle cases elsewhere.

Reply:
👍